### ISSUE TITLE
Accessibility: Narrator cannot announce PropertyBrowser top-level and sub-level items expanded/collapsed state

### ISSUE THREAD

**Issue by Marie-Shi** [dotnet/winforms#3159]
* .NET Core Version: 
.Net Core 5.0.100-preview.5.20228.7 from 5.0 master branch + VS2019 with latest core designer

* Have you experienced this same bug with .NET Framework?: 
Yes

**Problem description:**

PropertyBrowser top-level and sub-level items accessibility: Narrator cannot announce top-level and sub-level items expanded/collapsed state.

**Minimal repro:**
1.	Create a winforms core application.
2.	Start Narrator and focus on Font and Accessibility property in property browser window, observe Narrator behavior.
[WindowsFormsApp1.zip](https://github.com/dotnet/winforms/files/4550312/WindowsFormsApp1.zip)

**Actual behavior:**
Narrator cannot announce Font and Accessibility property expanded/collapsed state.
![Narrator_not_read_expand_collapsed](https://user-images.githubusercontent.com/45864017/80571756-bac83380-8a2f-11ea-90b9-ba07637efd70.gif)

**Expected behavior:**
Narrator should announce Font and Accessibility property expanded/collapsed state correctly.

**Comment by merriemcgaw** [comment]
@Marie-Shi - could you please file a .NET Framework bug for this too?
@M-Lipin - seems related to #3162

**Comment by Marie-Shi** [comment]
@merriemcgaw there is a .NET Framework bug [386116](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/386116) to track expand/collapse property in Property browser window, we have done private testing for it, issue is fixed, the details please see here: [Private testing for bug 386116](https://microsoft.sharepoint.com/:o:/r/teams/MerriesWinFormsandSetup/_layouts/15/WopiFrame2.aspx?sourcedoc={3418dda4-40a9-4626-8ae0-033052fa7a0b}&action=edit&wd=target%28Accessibility%20Testing%2Eone%7C8E831535%2D29F7%2D4400%2DA3D7%2D7C935D5FD6B1%2FRe%2Dtest%20on%204%5C%2F21%7CB9C13CFA%2D9C98%2D4134%2D9330%2D4701C6F8F109%2F%29)

**Comment by merriemcgaw** [comment]
Perfect! I had forgotten all about that one.
